Makefile

```
@@ -167,21 +200,49 @@ ginkgo-api7ee-e2e-test: adc
+		--organization="$(CONFORMANCE_ORGANIZATION)" \
+		--project="$(CONFORMANCE_PROJECT)" \
+		--url="$(CONFORMANCE_URL)" \
+		--version="$(CONFORMANCE_VERSION)" \
+		--contact="$(CONFORMANCE_CONTACT)" \
+		--mode="$(CONFORMANCE_MODE)" \
 		--report-output=$(CONFORMANCE_TEST_REPORT_OUTPUT)
 
 
 .PHONY: conformance-test-api7ee
+conformance-test-api7ee: export INGRESS_IMAGE=$(CONFORMANCE_INGRESS_IMAGE)
+conformance-test-api7ee: export ADC_IMAGE=$(CONFORMANCE_ADC_IMAGE)
```

checkmake (0.3.2)

[warning] Target "conformance-test-api7ee" defined multiple times (lines 225 and 226).

(uniquetargets)